templates/_layout.html

```diff
@@ -14,8 +14,8 @@
       <meta property="twitter:site" content="@snapcraftio" />
     {% endblock %}
 
-    <link rel="icon" type="image/png" href="https://assets.ubuntu.com/v1/fdc99abe-ico_16px.png" sizes="16x16" />
```

Comment:
Don't do this. It's much better to link people directly to the correct file than to send them to a different URL that then redirects.